ONSdigital/response-operations-ui
# RAS-1175 Remove disable-send.js from ROPs

# What and why?
disable-send.js is no longer used and double submission is now being blocked by the design system 
# How to test?
Make sure when sending a secure message that you can not send it twice, the DS loader should spin instead
# Jira
